### DISCUSSION TITLE
Calculator example, unary operators

### DISCUSSION THREAD

**Discussion by hia3** [taocpp/PEGTL#16] [Q&A] [answered]
Calculator example can't parse "-(5+5)".

**Comment by ColinH** [comment, marked as answer]
For simplicity, the calculator example doesn't currently implement unary operators (except for signs that are part of an integer literal).

**Comment by ColinH** [comment]
In case you opened this issue because you are interested in how to handle unary operators in the calculator example, which is not quite trivial, you can contact us at `pegtl(at)colin-hirsch.net`.

**Comment by hia3** [comment]
Thanks :-)

But I was just reading the example and thought that this might be omission. If this is as intended, I'm okay with that.
